docs/manual/system/settings.de.md

```diff
@@ -199,12 +199,11 @@ parameters:
 ## config.yml
 
 Die normale Bundle Config gehört in die `config.yml` und befindet sich im Ordner `app/config/`. 
-Falls die Datei noch nicht vorhanden ist, muss diese angelegt werden. Contao lädt automatisch die `config_prod.yml` 
-bzw. `config_dev.yml` und falls nicht vorhanden die `config.yml`.
+Falls die Datei noch nicht vorhanden ist, kann diese angelegt werden. Contao lädt automatisch die `config_prod.yml` 
+bzw. `config_dev.yml` und falls nicht vorhanden die `config.yml` und falls auch die nicht vorhanden die Standard-Konfiguration.
 
 Damit kannst du unterschiedliche Konfigurationen für deine Test- bzw. Produktionsumgebung (dev/prod) realisieren (z. B. 
-mehr Logging im Debug Modus). Außerdem committest du die `config.yml` im Gegensatz zur `parameters.yml` in dein 
-[Repository](https://de.wikipedia.org/wiki/Repository).
+mehr Logging im Debug Modus). 
```

Comment:
Das sollte auf jeden Fall _nicht_ entfernt werden. Nur weil du selbst deine Projekte nicht mit Git versionierst heißt das ja nicht, dass wir das deswegen für alle entfernen müssen.

Reply:
Wir müssen garnix...Hab ich auch nicht verlangt.

Nun gut, wenn man weiß, worum es hier geht (und das wird mir jetzt klarer), dann dankt man für diesen Hinweis. Alle Anderen fragen sich, ob Sie das verstehen müssen.  Vielleicht wäre ein Hinweis gut, dass dieser Hinweis für den Fall interessant ist, dass man seine Projekte z.B. mit Git versioniert.

Reply:
> Vielleicht wäre ein Hinweis gut, dass dieser Hinweis für den Fall interessant ist, dass man seine Projekte z.B. mit Git versioniert.

Dann am besten diesen Vorschlag in deinem PR bringen ;)